agent/proto/agent.proto

```diff
@@ -264,6 +266,29 @@ message BannerConfig {
 	string background_color = 3;
 }
 
+message WorkspaceAgentScriptCompletedRequest {
+    Timing timing = 1;
+}
+
+message WorkspaceAgentScriptCompletedResponse {
+}
+
+message Timing {
+    bytes script_id = 1;
+    string display_name = 2;
```

Comment:
What do we need display name for if ID is already given, isn't that available in the DB?

Reply:
I guess we don't need it although it does save a call to the DB as fetching the manifest passes along this `display_name` for us.

Reply:
The way I envision this behaving is that we only reference the script id in the timing/result table and when we pull out the data where we need the display name, we join the table with agent scripts to get the display name. The agent can still use the display name from manifest for logging and what not if that's useful, but doesn't have to waste bytes by sending it back.

Reply:
In that case I'm happy to remove this then. So that'd be `display_name` from the `workspace_agent_script_timings` table?

Reply:
Yeah, that's the one 👍🏻